### PR TITLE
Add local route to Simorgh for IDX data

### LINKED ISSUE #6432: IDX - Add local route to Simorgh for IDX data
**Is your feature request related to a problem? Please describe.**
For local development, we should add a route in Simorgh that allows us to develop against fixture data.

**Describe the solution you'd like**

- Fixture data can be added to the `data` folder for `persian/afghanistan`, and for `ukrainian/ukraine_in_russian`. This data should come from the ares feed for each of these IDX pages.

- We then need to add a regex for the data path for IDX pages - which will match the path specified for the pages in #6422, with `.json` appended at the end.

- We can add a local route in simorgh that can be developed against locally, taking https://github.com/bbc/simorgh/blob/c1f8432c89bfd8c1f0c410a9162dc69065222473/src/server/index.jsx#L169 as an example.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #6432

**Overall change:**
Added a route in Simorgh that allows us to locally develop against fixture data.
To check that this is working go to:
http://localhost:7080/persian/afghanistan.json
http://localhost:7080/ukrainian/ukraine_in_russian.json

**Code changes:**
- Added IDX fixure data from the Ares feed for `persian/afghanistan` and for `ukrainian/ukraine_in_russian`.
- Added regex for the data path for IDX pages.
- Added a local route for IDX.
- Added tests for checking if IDX path returns data.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/server/index.jsx [hunk 1]
```diff
@@ -65,6 +66,7 @@ const constructDataFilePath = ({
   switch (pageType) {
     case 'frontpage':
     case 'mostRead':
+    case 'idx':
```
Comment (tochwill) [inline src/server/index.jsx]: Don't think you need this in here as you're constructing the path below!

Comment (OlgaLyubin) [reply src/server/index.jsx]: True, removed this:)

File: src/app/routes/utils/regex/index.js [hunk 2]
```diff
@@ -53,3 +53,4 @@ export const secondaryColumnDataRegexPath = getSecondaryColumnDataRegex(
 );
 
 export const IdxPagePath = getIdxPageRegex();
+export const IdxDataRegexPath = `${IdxPagePath}.json`;
```
Comment (DenisHdz) [inline src/app/routes/utils/regex/index.js]:
```suggestion
export const IdxDataPath = `${IdxPagePath}.json`;
```

File: src/server/index.jsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ import {
   mostReadDataRegexPath,
   legacyAssetPageDataPath,
   secondaryColumnDataRegexPath,
+  IdxDataRegexPath,
```
Comment (DenisHdz) [inline src/server/index.jsx]:
```suggestion
  IdxDataPath,
```

File: src/server/index.jsx [hunk 4]
```diff
@@ -224,6 +225,11 @@ if (process.env.SIMORGH_APP_ENV === 'local') {
 
       sendDataFile(res, dataFilePath, next);
     })
+    .get(IdxDataRegexPath, async ({ params }, res, next) => {
```
Comment (DenisHdz) [inline src/server/index.jsx]:
```suggestion
    .get(IdxDataPath, async ({ params }, res, next) => {
```

**Review by DenisHdz — COMMENTED** [review]
Looks good! Just a suggestion to rename the path to be consistent with the other pages types in the file.

**Review by mmcculloch — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]

**Comment by PriyaKR** [comment]
LGTM.

**Pull request merged**